libglusterfs/src/inode.c

```diff
@@ -1589,7 +1590,7 @@ inode_table_prune(inode_table_t *table)
                     break;
                 }
             }
-
+            GF_ASSERT(table->lru_size);
```

Comment:
For inode specific changes we can handle from this patch https://github.com/gluster/glusterfs/pull/1776

Reply:
I would suggest please keep GF_ASSERT changes only specific to this patch.
For specific to inode code change either use #1776 or send a different patch.

Reply:
Okay. I will remove the inode specific changes and resubmit the patch